requirements-localprocessing.txt

```diff
@@ -1,5 +1,5 @@
 # Those requirements are for the experimental and optional client side processing feature
 # that can be installed via pip install openeo[localprocessing].
 # Please be aware that this functionality is not fully stable and may change quickly.
-openeo-pg-parser-networkx==2023.3.1
-openeo-processes-dask==2023.3.2
```

Comment:
I think we should eliminate `requirements-localprocessing.txt` and handle everything through the `localprocessing` extra of `setup.py`,

Reply:
the docs already instruct to do
```
pip install openeo[localprocessing]
```

anyway